packages/browser-wallet/src/popup/pages/Account/i18n/da.ts

```diff
@@ -210,15 +209,117 @@ const t: typeof en = {
                 descriptionRedelegate:
                     'Vil du gerne automatisk indsætte dine fortjenester på din delegation saldo?\n\nHvis du vælger ikke at indsætte dine fortjenester, vil de blive tilføjet til din offentlige saldo hver pay day.',
                 locked: 'Saldo låst',
-                lockedNote: 'Du kan ikke ændre saldoen mens der er en ændring kommende ændring',
+                lockedNote: 'Du kan ikke ændre saldoen mens der er en kommende ændring',
                 overStakeThresholdWarning:
-                    'Du er ved at låse mere end {{ threshold }}% af din totale saldo som diun delegation saldo.\n\nHvis du ikke har nok ulåste CCD, vil du ikke kunne betale for transaktion omkostninger.',
+                    'Du er ved at låse mere end {{ threshold }}% af din totale saldo som din delegation saldo.\n\nHvis du ikke har nok ulåste CCD, vil du ikke kunne betale for transaktion omkostninger.',
                 enterNewStake: 'Indtast ny delegation saldo',
             },
             continueButton: 'Fortsæt',
             warning: 'Advarsel',
         },
     },
+    baking: {
+        registerIntro: {
+            '1': {
+                title: 'Opret en baker',
+                body: 'En baker er en node som deltager i netværket ved at bake (lave) nye blocks som bliver tilføjet til kæden. Hver baker har et nogle kryptogragiske nøgler, som refereres til som "baker keys", som noden bruger når den baker blocks. Du genererer baker nøgler når du tilføjer en baker konto. Når baker noden er blevet genstartet med baker keys, så den starte med at bake, to epochs efter transaktionen er blevet godkendt.',
```

Comment:
```suggestion
                body: 'En baker er en node som deltager i netværket ved at bake (lave) nye blocks som bliver tilføjet til kæden. Hver baker har et nogle kryptogragiske nøgler, som refereres til som "baker keys", som noden bruger når den baker blocks. Du genererer bakernøgler når du tilføjer en bakerkonto. Når baker noden er blevet genstartet med bakernøglerne, så vil den starte med at bake to epochs efter transaktionen er blevet godkendt.',
```

What do you mean by "de kryptografiske nøgler refereres til som "baker keys""? I think we should either write "baker keys" everywhere, or "bakernøgler" everywhere.

Personally I think "bakernøgler" makes the most sense with the previous reference to "kryptografiske nøgler"?

Reply:
I think it makes sense to say "baker keys". :thinking: